src/components/molecules/VNavigation.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <nav class="nav">
+    <ul class="nav__ul">
+      <li>
+        <router-link to="/" exact>
+          <v-smart-home class="nav__icon" />
+          <span :class="isActive('/')"></span>
+        </router-link>
+      </li>
+      <li>
+        <router-link to="" exact>
+          <v-calendar-time class="nav__icon" />
+          <span :class="isActive('')"></span>
+        </router-link>
+      </li>
+      <li>
+        <router-link to="" exact>
+          <v-chart-area-line class="nav__icon" />
+          <span :class="isActive('')"></span>
+        </router-link>
+      </li>
+    </ul>
+  </nav>
+</template>
+
+<script>
+import {
+  VSmartHome,
+  VCalendarTime,
+  VChartAreaLine,
```

Comment:
what if you rename your icons name become like this?

```suggestion
  VSmartHomeIcon,
  VCalendarTimeIcon,
  VChartAreaLineIcon,
```

Doing so will make you able to differentiate the icon components among the others in `<template>` tag
```
<template>
  <div>
    <not-icon-component />

    <v-smart-home-icon />
    <v-calendar-icon />
    <v-chart-area-line-icon />
  </div>
</template>
```

Reply:
got it